buildSrc/src/main/java/io/micronaut/guides/core/Guide.java

```diff
@@ -0,0 +1,96 @@
+package io.micronaut.guides.core;
+
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.jsonschema.JsonSchema;
+import io.micronaut.serde.annotation.Serdeable;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+@JsonSchema
+@Serdeable
+public record Guide(
+        @NonNull
+        @JsonPropertyDescription("The guide's title")
+        String title,
+
+        @NonNull
+        @JsonPropertyDescription("The guide introduction")
+        String intro,
+
+        @NonNull
+        @JsonPropertyDescription("The guide's authors")
+        List<String> authors,
+
+        @NonNull
+        @JsonPropertyDescription("The guide's categories")
+        List<String> categories,
+
+        @NonNull
+        @JsonPropertyDescription("The guide publication date. It should follow the format YYYY-MM-DD")
+        LocalDate publicationDate,
+
+        @JsonPropertyDescription("If the guide needs a minimum Java version, define it here")
+        Integer minimumJavaVersion,
+
+        @JsonPropertyDescription("If the guide needs a maximum Java version, define it here")
+        Integer maximumJavaVersion,
+
+        @JsonPropertyDescription("The acronym for the cloud service provider of the guide. For example, OCI for Oracle Cloud Infrastructure")
+        Cloud cloud,
+
+        @JsonPropertyDescription("Set it to true to skip running the tests for the Gradle applications for the guide")
+        Boolean skipGradleTests,
```

Comment:
use the primitive `boolean` instead which defaults to `false`. The default for this property.